tests/unit/resources/standalone-help.txt

```diff
@@ -11,6 +11,9 @@ Options:
                         password for use with authentication and/or sudo
 
 
+The commands listed below are for the current mode.
+The current mode can be changed using the 'mode select' command.
```

Comment:
You probably should list the two possible modes here, and maybe also say what mode the user is currently in.